bittide-instances/src/Bittide/Instances/Pnr/ProcessingElement.hs

```diff
@@ -22,20 +25,35 @@ import Bittide.SharedTypes
 import Bittide.Wishbone
 import Project.FilePath
 
+alwaysAck :: KnownNat nBytes => Circuit (Wishbone dom 'Standard addrW (Bytes nBytes)) ()
+alwaysAck = Circuit go
+ where
+  go (_m2s, ()) = (pure emptyWishboneS2M{acknowledge=True}, ())
+
```

Comment:
unless I'm missing it, this function doesn't seem to be used in _this file_, right?

Is the always ACK really what's needed? I would have to check the Wishbone spec, but shouldn't it check `CYC + STB` before giving a termination signal?

Reply:
> I would have to check the Wishbone spec, but shouldn't it check CYC + STB before giving a termination signal?

I recently read in the wishbone spec that you indeed need to account for CYC/STB!

I think `alwaysAck` is a leftover though, so we can remove it entirely.